packages/mesos-modules/buildinfo.json

```diff
@@ -3,7 +3,7 @@
     "single_source" : {
       "kind": "git",
       "git": "https://github.com/dcos/dcos-mesos-modules.git",
-      "ref": "8d09c2d44f606df23a57254ebcbc5f2722a3b677",
-      "ref_origin": "master"
+      "ref": "86f14f9402e324f78385a0b7b9509079dda7c0ee",
+      "ref_origin": "1.8.x"
```

Comment:
@adam-mesos I see the systemd references in README.md, but I thought removed all references to systemd and journald headers from `configure.ac`? Could you point to any that you might have found in `configure.ac`?